webrtc.html

```diff
@@ -689,14 +689,10 @@
         </ol>
         <p>To <dfn id="set-description" data-lt="set the RTCSessionDescription">set an RTCSessionDescription</dfn>
         <var>description</var> on an <code><a>RTCPeerConnection</a></code>
-        object <var>connection</var>, run the following steps:</p>
```

Comment:
(commenting on existing wording two lines above this one, which relates to changes later in this patch)

"RTCSessionDescription" is used as a descriptive name for the *description* input parameter here, as opposed to describing the target being set. This makes the meaning (and wording, "the" vs "an") subtly opposite of the text where we link from.

Would it be better to say "To **set the RTCSessionDescription** to a *description*" ?